tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/generalize_zip_codes_test.py

```diff
@@ -0,0 +1,143 @@
+"""
+Integration test for generalize_zip_codes module
+
+This cleaning rule generalizes all PPI concepts containing zip codes to their
+primary three digit representation, (e.g. 35400 → 354**).
+
+This cleaning rule is specific to the controlled tier.
+
+
+Original Issue: DC1376
+"""
```

Comment:
Update this to mention that the zip code will be obscured only if `observation_source_concept_id` is 1585250